spec/requests/api/v1/allocations_controller_index_spec.rb

```diff
@@ -167,6 +167,50 @@
           expect(response.body).to eq('{"error":{"locations":["may not be used in combination with `from_locations` or `to_locations` filters."]}}')
         end
       end
+
+      describe 'included relationships', :skip_before do
+        let!(:allocations) { create_list :allocation, 2, :with_moves }
+
+        before do
+          get "/api/v1/allocations#{query_params}", params: params, headers: headers
```

Comment:
I think we only need to pass `params` here as those are converted to query params?

Reply:
I just preferred the explicit query formulation. I know you're technically correct and that it all gets bundled together. If you feel strongly about it I can put the query into the params

Reply:
So I validated this by passing `foo: bar` in the params hash and seeing whats loaded in the controller action. These values get merged in request specs properly :+1:.

Reply:
Not a blocker: I would prefer if we use `params` to avoid having to encode and format the query params ourselves. If we go with `query_params` however, can you get rid of `params: params` as we aren't using it in this context, and main hash is empty:
https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/483/files#diff-ca0a918cadf1b72e0d225d2948b64e0eL17
so something like this: 
```
get "/api/v1/allocations#{query_params}", headers: headers
```